test/autotest.py

```diff
@@ -753,7 +753,7 @@ def saveResultsNMI():
 # program will try to unlink the file once again, but the unlink operation will
 # fail, causing the test to fail.
 old_ckpt_cmd = CKPT_CMD
-CKPT_CMD = 'xc'
+CKPT_CMD = b'bc'
```

Comment:
Isn't 'bc' for "blocking until done"? And 'xc' is for "kill after checkpoint"? Has the behavior changed recently?

(I just checked `dmtcp_command.cpp` and the 'xc' command is commented out. So it seems like we don't support that anymore.

Also, the documentation for 'bc' in `dmtcp_command.cpp` (line 52) says:

> -bc, --bcheckpoint:    Checkpoint all nodes, blocking until done

If this is true, then then test would fail after this change. According to the comment on line 751 above, the program would unlink the temp file after resuming from checkpoint, and then the unlink operation would fail on restart, causing the test to fail.)

Reply:
I looked at when 'xc' and 'bc' were introduced.  It was around 2013.  I think the intention was that it was originally 'xc', and we wanted to change the name to 'bc', but I'm not sure.  In any case, from the beginning, `src/dmtcp_command.cpp` shows that these two commands were always synonyms, and never had different effects.

So, I think we're donig a late cleanup here, for somethign that happened in 2013.

Reply:
@rohgarg, You're correct in all of the above.  See my newer description in this PR now.